src/legacy/core_plugins/kibana/public/visualize/plugin.ts

```diff
@@ -154,8 +149,6 @@ export class VisualizePlugin implements Plugin {
       showOnHomePage: true,
       category: FeatureCatalogueCategory.DATA,
     });
-
-    VisEditorTypesRegistryProvider.register(defaultEditor);
```

Comment:
This looks like it isn't allowed to use `editor: 'default'` anymore. We should also make this part of the dev docs. It seems like the default editor is also the default value of `editor`, so the suggested migration would be to remove `editor:` from the visualization definition, right?

Reply:
You are partially right. The default editor was used by default previously in `src/legacy/core_plugins/visualizations/public/np_ready/public/types/base_vis_type.js`.
But you are right, the default editor controller is not registered now with `VisEditorTypesRegistryProvider`. So if any vis type has this property, it should be removed.

Reply:
OK, could you add a second section to the Dev Docs explaining the change in this case? Just to make sure plugin authors can follow.

Reply:
Done. But I'm not sure if I made it in a right way, could you please modify it if needed?

Reply:
That's fine 👍